### ISSUE TITLE
Elasticsearch didn't start during :example-plugins:script-expert-scoring:yamlRestTest (elasticsearch-master-g1gc) reporting: Multiple garbage collectors selected

### ISSUE THREAD

**Issue by dliappis** [elastic/elasticsearch#63611]
Occurs with $RUNTIME_JAVA_HOME < 15.

**Build scan**: https://gradle-enterprise.elastic.co/s/ou63qq3svaoxa

**Repro line**: `./gradlew -Druntime.java=11 -Dtests.seed=78D1A8483F9345AD ':rest-api-spec:yamlRestTest'   --tests "org.elasticsearch.test.rest.ClientYamlTestSuiteIT"   -Dtests.method="test {p0=cat.segments/10_basic/Help}" -Dtests.jvm.argline='-XX:-UseSerialGC -XX:+UseG1GC'`

(note that the java argline is forced by the job https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+g1gc)

**Reproduces locally?**: Yes

**Applicable branches**: master

**Failure history**: https://gradle-enterprise.elastic.co/s/ohlpfrdcnuqno / https://gradle-enterprise.elastic.co/s/g3jkimrg2oxli / https://gradle-enterprise.elastic.co/s/miak6gk65y3ke / https://gradle-enterprise.elastic.co/s/relzi6rd7nc5s

**Failure excerpt**:

```
Task failed with an exception.
-----------
* What went wrong:
 Execution failed for task ':example-plugins:rest-handler:javaRestTest'.
 > process was found dead while waiting for ports files, node{:example-plugins:rest-handler:javaRestTest-0} 
```

but looking at the uploaded server logs we see under `rest-api-spec/build/testclusters/yamlRestTest-0/logs/es-stderr.log`:

```
Exception in thread "main" java.lang.RuntimeException: starting java failed with [1]
output:
Error occurred during initialization of VM
Multiple garbage collectors selected
error:
OpenJDK 64-Bit Server VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future relea
se.
        at org.elasticsearch.tools.launchers.JvmErgonomics.flagsFinal(JvmErgonomics.java:143)
        at org.elasticsearch.tools.launchers.JvmErgonomics.finalJvmOptions(JvmErgonomics.java:108)
        at org.elasticsearch.tools.launchers.JvmErgonomics.choose(JvmErgonomics.java:56)
        at org.elasticsearch.tools.launchers.JvmOptionsParser.jvmOptions(JvmOptionsParser.java:138)
        at org.elasticsearch.tools.launchers.JvmOptionsParser.main(JvmOptionsParser.java:96)
```

**Comment by rjernst** [comment]
@breskeby Can you take a look at this? Seems it could be related to https://github.com/elastic/elasticsearch/issues/62716

**Comment by breskeby** [comment]
I provided a PR with a fix and a further explanation at https://github.com/elastic/infra/pull/24210 basically `mea culpa` as this issue was introduced by myself by trying to fix #62716

**Comment by breskeby** [comment]
closing as this JOB was removed by @mark-vieira as we identified that job to be superfluous superfluous
